### PR TITLE
collect-info.sh: add debug info from ModemManager

### PR DESCRIPTION
**Pull request by milan-zededa**
Even though we already collect pubsub messages published by mmagent from the wwan microservice (WwanStatus, WwanMetrics, WwanLocationInfo), it can still be useful for troubleshooting to collect detailed (and some additional) info about modems, SIMs and bearers directly from the ModemManager.

For this to work I had to enable running `eve exec` with process forked by nsenter, see the first commit description.

### CODE REVIEW

**Review by uncleDecart — APPROVED** [review]
LGTM, bash-ninja collection in EVE is improving :D

File: pkg/debug/scripts/collect-info.sh [hunk 1]
```diff
@@ -155,6 +155,36 @@ collect_network_info()
...
+    MODEMS="$(eve exec --fork wwan mmcli -L |\
+              sed -n 's/.*\/ModemManager1\/Modem\/\([0-9]\+\).*/\1/p' | uniq)"
+    for MODEM in $MODEMS; do
+        INFO="$(eve exec --fork wwan mmcli -m "$MODEM")"
+        echo
+        echo "Modem $MODEM:"
+        echo "$INFO"
+        SIMS="$(echo "$INFO" |\
+                sed -n 's/.*\/ModemManager1\/SIM\/\([0-9]\+\).*/\1/p' | uniq)"
+        for SIM in $SIMS; do
```
Comment (rene) [inline pkg/debug/scripts/collect-info.sh:168]: Doesn't make sense to put a message here if no SIM card is inserted?

Comment (milan-zededa) [reply pkg/debug/scripts/collect-info.sh:168]: No need, this information will be obvious from `mmcli -m "$MODEM"` (aka `echo $INFO` in the script)

Comment (milan-zededa) [reply pkg/debug/scripts/collect-info.sh:168]: Notice "failed reason: sim-missing"

File: pkg/dom0-ztools/rootfs/bin/eve [hunk 2]
```diff
@@ -8,7 +8,7 @@ help() {
 Welcome to EVE!
   commands: enter [service] [command (assumed sh)]
             enter-user-app <service>
-            exec [service] command
+            exec [--fork] [service] command
```
Comment (rouming) [inline pkg/dom0-ztools/rootfs/bin/eve:11]: I see option '--fork' but not '--no-fork' as commit msg says.

**Review by rouming — APPROVED** [review]
Ack

File: pkg/dom0-ztools/rootfs/bin/eve (hunk 2 above)
Comment (milan-zededa) [reply pkg/dom0-ztools/rootfs/bin/eve:11]: Fixed

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**